extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -118,13 +118,24 @@ export class CookieWriter {
       return Promise.resolve();
     }
 
+    // convert cookieMaxAge (sec) to milliseconds
+    const cookieExpireDateMs = hasOwn(inputConfig, 'cookieMaxAge')
```

Comment:
How do we want to handle invalid `cookieMaxAge` value? 
`cookieMaxAge = '1234'` or `cookieMaxAge = 'invalid'`

There's the `#isValidCookieConfig_` where I think is the right place to add assertion

Reply:
I dont think `isValidCookieConfig_` is the best place for this, this function is used to check validity of cookie values. There's no overlap between that and validity of `cookieMaxAge`. I think adding a new method `getCookieMaxAge_(config)` would be cleaner.

Also, looking at how the rest of the code handles invalid numbers in the config, they do something like `Number(spec['totalTimeMin']) || 0`, trying to convert it to a number first then providing a default value if invalid.

Reply:
I am seeing the `#isValidCookieConfig_` function as the place to check the validity of the entire configuration. But I think you make a good point on we don't want assertion, instead we want to fallback to a default value. 
`#getCookieMaxAge_()` works! Just one more request. If the value is not valid. Please report it as an user error, indicating that we expiration time falls back to default value due to invalid config. Thanks.